### ISSUE TITLE
Pattern:  Content block Media - Accessibility- Screen reader - For Image links, provide a descriptive link text or aria-label or alt text.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1425]
## Detailed description
Device / browser - Windows Firefox (JAWS), iPhone VO safari

Issue:- For Image links, either provide a descriptive link text or aria-label or alt text.

Suggestion :  Informative graphics should be given a descriptive alternative text so that the users of assistive technology can understand their purpose.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/318dd187-bb96-4987-952b-ab4b1f391b50)

**Comment by RobertaJHahn** [comment]
The description text is provided by the people writing the content.  The alt text is provided for the content authors to provide more descriptive content.  

@kennylam can you check to be sure this is handled correctly. Then close.

**Comment by kennylam** [comment]
> @kennylam can you check to be sure this is handled correctly. Then close.

@RobertaJHahn Alt text is present in this component's image. Closing
